src/cairoUtilFuncGen/storage/dynArrayPushWithArg.ts

```diff
@@ -144,40 +145,34 @@ export class DynArrayPushWithArgGen extends StringIndexedFuncGen {
     const arrayName = dynArray.name;
     const lengthName = dynArrayLength.name;
     const funcName = `${arrayName}_PUSHV${this.generatedFunctionsDef.size}`;
-    const implicits =
-      argLoc === DataLocation.Memory
-        ? '{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr : felt, warp_memory: DictAccess*}'
-        : '{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr : felt, bitwise_ptr: BitwiseBuiltin*}';
+    const implicit = argLoc === DataLocation.Memory ? '#[implicit(warp_memory)]' : '';
```

Comment:
Update the implicit syntax

Reply:
Can you please specify the new decorator syntax for warp-memory implicit

Reply:
Rodrigo meant to add type annotation I guess